frontend/src/components/FundsPullPreApproval/FundsPullPreApproval.tsx

```diff
@@ -0,0 +1,83 @@
+import { Approval } from "../../interfaces/approval";
+import React from "react";
+import { classNames } from "../../utils/class-names";
+import { Button, Col, Container, Row } from "reactstrap";
+import ApprovalDetails from "./ApprovalDetails";
+
+interface ApprovalProps {
+  approval: Approval;
+  onApproveClick: () => void;
+  onRejectClick: () => void;
+  onRevokeClick: () => void;
+  onAnyClickSetApproval: () => void;
+  displayApproveRejectButtons: boolean;
+  displayRevokeButton: boolean;
+}
+
+function FundsPullPreApproval({
+  approval,
+  onApproveClick,
+  onRejectClick,
+  onRevokeClick,
+  onAnyClickSetApproval,
+  displayApproveRejectButtons,
+  displayRevokeButton,
+}: ApprovalProps) {
+  const itemStyles = {
+    "list-group-item": true,
```

Comment:
This value is constant, you probably don't need this object and `classNames` anymore